tests/preprocessor/include-search-path.slang

```diff
@@ -0,0 +1,13 @@
+//TEST:SIMPLE: -Itests/preprocessor/include
```

Comment:
Thank you for adding a test case for this. No matter what we can have some confidence I won't mess this up again (or at least not in the same way). :)